common/buildcraft/silicon/tile/TileLaserTableBase.java

```diff
@@ -45,6 +45,8 @@ protected TileLaserTableBase() {
 
     protected abstract long getTarget();
 
+    public final long getTarget2() {return getTarget();}
```

Comment:
euh what's the point of this method? you might as well just make getTarget() public

Reply:
The only reason I can think of is to keep compatibility with older subclasses or to minimise changes... neither of which are really relevant.

Reply:
yeah, i highly doubt anyone is extending this, the small chanse of breaking a theoretical addon isn't worth adding this method i think